src/simplejavacalculator/BufferedImageCustom.java

```diff
@@ -14,7 +14,7 @@ public Image imageReturn()
       throws IOException {
       Image image;
       
-      InputStream bis = getClass().getResourceAsStream("/icon/icon.png");
+      InputStream bis = getClass().getResourceAsStream("/Resources/icon/icon.png");

```

Comment:
Shouldn't `Resources/` be lowercase (`resources/`)?

Reply:
Yes, it should be.

Reply:
Because in your code, it isn't. The first letter (`R`) is capitalized :)

Reply:
Still not sure how you got this error by the way. Are you running the project from the root directory? Normally, you shouldn't get any problem with the icon located in `/icon/ folder outside of the `src` folder.